package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "lint": "eslint src --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint": "eslint . --fix --max-warnings=0",
+    "prettier:fix": "prettier . --write",
```

Comment:
Would you consider combining the two commands on line 9 & 10 to perform prettier and lint fixes in one go. Refer to an example below:

```json
"prettier:fix": "prettier . --write",
"lint": "npm run pretter:fix eslint . --fix --max-warnings=0",
```

Same as the `test` and `coverage` commands are combined.